upgrade-notes/backend_20240726_093001.md

```diff
@@ -0,0 +1,167 @@
+#### product files support ([#3288](https://github.com/shopsys/shopsys/pull/3288))
+
+-   Filesystem directory structure changed for `UploadedFile` has changed. Files are now stored in parent directory directly and not in subdirectories by entity name.
+    Migration command `\Shopsys\FrameworkBundle\Command\MigrateUploadedFilesCommand` is provided to move files to the new structure and runs automatically
+    as part phing target `build-deploy-part-2-db-dependent`. Make sure to back up all uploaded files before deploying this change as it may result in data loss.
```

Comment:
```suggestion
    as a part of phing target `build-deploy-part-2-db-dependent`. Make sure to back up all uploaded files before deploying this change as it may result in data loss.
```